### ISSUE TITLE
Unable to evaluate pull request

### ISSUE THREAD

**Issue by shamikct** [lerebear/sizeup-action#64]
@lerebear Thank you for fixing the earlier bug.

This is a continuation of this #62.

I have made the necessary changes in my yml, and i am getting another error: 
<img width="548" alt="image" src="https://github.com/user-attachments/assets/65801f6c-9f3c-43f1-a369-2b1cd8d6edd1">

Error:
```python
Run lerebear/sizeup-action@v0.8.9
  with:
    token: ***
    git-diff-options: --ignore-space-change
    configuration-file-path: .github/workflows/sizeup/config.yml
Reading sizeup configuration from ".github/workflows/sizeup/config.yml"
Evaluating pull request codeandtheory/ctailml#158
Error: Cannot read properties of undefined (reading 'match')
```

**Comment by lerebear** [comment]
Thanks again for the report @shamikct. Could you please share the contents of the config file you have at `.github/workflows/sizeup/config.yml` on the branch of the pull request you are trying to evaluate?

**Comment by shamikct** [comment]
Here you go.

```yaml
labeling:

  # Whether or not to add a label to each pull request to indicate its assessed category
  applyCategoryLabels: true

  # The prefix to add to each category label that we apply.
  categoryLabelPrefix: "pr_sizeup/"

commenting:

  # Whether or not to comment on pull requests that exceed the configured score threshold
  addCommentWhenScoreThresholdHasBeenExceeded: true

  # The threshold above which we will add a comment to the assessed pull request.
  scoreThreshold: 500

  # The template for the comment that should be added to each pull request that
  # exceeds the score threshold. Any of the following variables can be included
  # in the template with surrounding curly braces (e.g. {{author}}) in order to
  # interpolate a computed value into the comment:
  #
  #   - author
  #   - threshold
  #   - score
  #   - category
  #   - score-details
  commentTemplate: |
    👋 @{{author}} this pull request exceeds the configured reviewability score threshold of {{threshold}}. Your actual score was {{score}}.
    The PR category is {{category}}.

# Configuration for how to evaluate pull requests.
# This is of the same format that `sizeup-core` accepts directly.
sizeup:
  categories:
    - name: extra small
      lte: 10
      label:
        name: xs
        color: 3cbf00
    - name: small
      lte: 30
      label:
        name: s
        color: 5d9801
    - name: medium
      lte: 100
      label:
        name: m
        color: 7f7203
    - name: large
      lte: 500
      label:
        name: l
        color: a14c05
    - name: extra large
      label:
        name: xl
        color: c32607
  ignoredFilePatterns:
    - CODEOWNERS
    - SERVICEOWNERS
  testFilePatterns:
    - "*_test.rb"
  scoring:
    formula: "- - + additions deletions"


```

**Comment by lerebear** [comment]
@shamikct I think this error is due to an invalid `formula` configuration. Rather than:

```yaml
  scoring:
    formula: "- - + additions deletions"
```

I think you need the following (which removes unmatched subtraction operators at the beginning of the formula):

```yaml
  scoring:
    formula: "+ additions deletions"
```

In the newly released version [`0.8.10`](https://github.com/lerebear/sizeup-action/releases/tag/v0.8.10) I introduced a more helpful error message for this situation, but correcting the formula as described above (without an upgrade) should be sufficient to fix the problem.

**Comment by shamikct** [comment]
Thanks @lerebear
